stable/distribution/ci/test-values.yaml

```diff
@@ -1,31 +1,20 @@
 # CI values for Distribution
 
-# replicaCount: 2
-# rbac:
-#   create: false
-#
-# serviceAccount:
-#   create: false
-
+databaseUpgradeReady: true
 postgresql:
   postgresPassword: password
   persistence:
     enabled: false
 
 redis:
-  password: "password"
+  password: password
   persistence:
     enabled: false
 
 distributor:
   persistence:
     enabled: false
-  loggers:
```

Comment:
Why did you remove the loggers?

Reply:
wanted to add these loggers support later - when we add it for unified charts